### PR TITLE
This closes #4, and closes #5

### LINKED ISSUE #4: Move mongo path from .env file to docker-compose config file

### PR DESCRIPTION
**Pull request by goodstemy**

### CODE REVIEW

**Comment by chebykin** [comment]
@goodstemy please, update the PR title with issue ids you resolving

**Comment by chebykin** [comment]
nope, not this way, just check out this doc https://help.github.com/articles/closing-issues-using-keywords/

File: monitor-service.yml [hunk 1]
```diff
@@ -54,5 +58,3 @@ networks:
       driver: default
       config:
       - subnet: 172.16.1.1/24
```
Comment (chebykin) [inline monitor-service.yml]: Why do we need this explicit subnet definition?

File: common.env [hunk 2]
```diff
@@ -1,2 +1 @@
 HTTP_PROVIDER_URI=http://docker.for.mac.localhost:8545
```
Comment (chebykin) [inline common.env]: This is a confusing name, change it to `WEB3_HTTP_PROVIDER_URI`

**Comment by chebykin** [comment]
@goodstemy ping

File: monitor-service.yml (hunk 1 above)
Comment (chebykin) [reply monitor-service.yml]: I think the default driver would be used by default, event without explicit specification

File: common.env (hunk 2 above)
Comment (goodstemy) [reply common.env]: Ok, I've changed name

File: monitor-service.yml (hunk 1 above)
Comment (goodstemy) [reply monitor-service.yml]: Rudimentary part of network config, removed.

File: common.env [hunk 3]
```diff
@@ -1,2 +1 @@
-HTTP_PROVIDER_URI=http://docker.for.mac.localhost:8545
-MONGO_URI=mongodb://mongo:27017/blockDB
\ No newline at end of file
+WEB3_HTTP_PROVIDER_URI=http://docker.for.mac.localhost:8545
```
Comment (chebykin) [inline common.env]: @goodstemy Are you sure that's it?

File: monitor-service.yml [hunk 4]
```diff
@@ -45,14 +49,10 @@ services:
...
     networks:
-      - backend
+      - app_net
 
 networks:
   app_net:
-    driver: bridge
+    driver: overlay
     ipam:
       driver: default
```
Comment (chebykin) [inline monitor-service.yml]: Have you tested this?

Comment (chebykin) [reply monitor-service.yml]: Why you choose overlay?

Comment (goodstemy) [reply monitor-service.yml]: Yeap.

File: common.env (hunk 3 above)
Comment (goodstemy) [reply common.env]: I can change hostname to testnet53.kenig.pw.

Comment (chebykin) [reply common.env]: oh, skip this since we decided to replace this explorer with another one

File: monitor-service.yml (hunk 4 above)
Comment (chebykin) [reply monitor-service.yml]: There should be no any network details at all, just the name

File: common.env (hunk 3 above)
Comment (goodstemy) [reply common.env]: Ok.

File: monitor-service.yml (hunk 4 above)
Comment (goodstemy) [reply monitor-service.yml]: Driver and other config parts besides name is not necessary.

**Pull request merged**